Library/Homebrew/cmd/info.rb

```diff
@@ -303,6 +306,17 @@ def info_formula(formula)
         ]
         if kegs.empty?
           puts "Not installed"
+          if args.github_manifest? && !(bottle = formula.bottle).nil?
+            begin
+              bottle.fetch_tab(quiet: !args.debug?)
+              bottle_size = bottle.bottle_size
+              installed_size = bottle.installed_size
+              puts "Bottle Size: #{disk_usage_readable(bottle_size)}" if bottle_size
+              puts "Installed Size: #{disk_usage_readable(installed_size)}" if installed_size
```

Comment:
Was looking into JSON API part (#18403) and I think we can just output size info if it is available. Can keep an argument to allow falling back on fetching manifest if info isn't locally available.